data-raw/standard_format_markdowns/combine_data_and_eda.Rmd

```diff
@@ -0,0 +1,251 @@
+---
+title: "Prey Data Overview"
+author: "Maddee Rubenson (FlowWest)"
+date: "`r Sys.Date()`"
+output: rmarkdown::github_document
+---
+
+```{r setup, include=FALSE}
+knitr::opts_chunk$set(echo = TRUE,
+                      warning = FALSE, 
+                      message = FALSE)
+
+library(tidyverse)
+library(lubridate)
+library(knitr)
+library(readxl)
+library(sf)
+library(preyDataProcessing)
+
+options(scipen=999)
+```
+
+## Prey Density 
+
+### Aggregate All Prey Data 
+
+```{r}
+
+all_prey_density <- bind_rows(preyDataProcessing::cordoleani_prey_data,
+                      preyDataProcessing::zeug_merced_prey_data, 
+                      preyDataProcessing::zeug_sanJoaquin_prey_data, 
+                      preyDataProcessing::montgomery_prey_data, 
+                      preyDataProcessing::guignard_prey_data, 
+                      preyDataProcessing::corline_prey_data,
+                      preyDataProcessing::zooper_prey_data) 
+
+# usethis::use_data(all_prey_density, overwrite = TRUE)
```

Comment:
Why is this commented out? If we rerun after changing data we will want to rebuild data

Reply:
same below